vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/dao/mysql/ClassData.java

```diff
@@ -77,7 +78,7 @@ public void add(AppClass appClass) throws DAOException {
             ps.close();
 
         } catch (SQLException ex) {
-            if (ex.getMessage().contains("Duplicate entry")) {
+            if (ex.getMessage().contains("Unique index or primary key violation")) {
```

Comment:
I guess the message error has changed ?
This is tricky as this could be linked to the database used.
I should make a test on a mysql to verify.